include/hobbes/net.H

```diff
@@ -324,7 +324,7 @@ _HNET_DEFINE_PRIMTYS(int32_t,  "int");
 _HNET_DEFINE_PRIMTYS(uint32_t, "int");
 _HNET_DEFINE_PRIMTYS(int64_t,  "long");
 _HNET_DEFINE_PRIMTYS(uint64_t, "long");
-_HNET_DEFINE_PRIMTYS(size_t,   "long");
+//_HNET_DEFINE_PRIMTYS(size_t,   "long");
```

Comment:
This is the llvm version used by travis-ci,  i
Apple LLVM version 8.0.0 (clang-800.0.38)
Target: x86_64-apple-darwin15.6.0
Thread model: posix